### ISSUE TITLE
Way to change perms for initializeNixDatabase

### ISSUE THREAD

**Issue by guskovd** [nlewo/nix2container#142]
Good afternoon. I would like to be able to manage `customizationLayer`. In particular, I would like to change the value of `mode` to `775` so that users of the group can use the same image

Now this value is hardcoded: https://github.com/nlewo/nix2container/blob/3853e5caf9ad24103b13aa6e0e8bcebb47649fe4/default.nix#L426
and I have no idea how to change it...

**Comment by nlewo** [comment]
@guskovd I don't think it is currently possible but we could add the argument `nixMode` (such as already existing `nixUid/nixGid` arguements).

**Comment by guskovd** [comment]
> @guskovd I don't think it is currently possible but we could add the argument `nixMode` (such as already existing `nixUid/nixGid` arguements).

That would be great!
